website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -155,4 +155,47 @@ Row placement determines the visual styling based on where the row is placed rel
 - We recommend using `medium` cell density by default.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
 - If content is largely string/text-based, `short` allows for more content to be displayed within the page.
-- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
\ No newline at end of file
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+## Multi-Selection
+
+Multi-selection enables users to select multiple rows in a table, enhancing efficiency and minimizing errors. This feature facilitates bulk operations, boosting productivity and optimizing data management.
+
+To enable multi-selection in a table:
+
+1. `Header Column / Selection` Component:
+Position it in the table's header row. This acts as a master control, enabling you to either select all rows simultaneously or deselect them.
+
+
+![Example of centered content within a table](/assets/components/table/multi-select-header.png)
+
+2. `Cell / Selection` Component:
+Integrate this into each table row. It allows for individual row selection and aligns with the Header Column component, reflecting the overall selection status of the table.
+
+![Example of centered content within a table](/assets/components/table/multi-select-cells.png)
+
+!!! Info 
+
+In the Ember component, unlike the `cell/select` figma component, achieving an indeterminate state is not possible. 
+!!!
+
+<video width="100%" controls loop>
+  <source
+    src="/assets/components/table/multi-selection-interaction-01.mp4"
+    type="video/mp4"
+  />
+</video>
+
+### Intended interaction
```

Comment:
I think this section might need a bit more fleshing out. Maybe consider breaking it down into a couple of different sections, could look something like this:

- Intended interaction
    - Within a simple table (this is where you can discuss what will happen with the selection states if the table is a single page/doesn't use pagination).
    - Within a paginated table
    - (maybe something else here?)